<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeIOWidget</class>
 <widget class="QWidget" name="VolumeIOWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume IO</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_HeatMap">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>热力图</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_HeatMap">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>当前Z</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QLabel" name="label_HeatMapZ">
             <property name="text">
              <string>--</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_MinZ">
               <property name="text">
                <string>--</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_HeatMapZ">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_MaxZ">
               <property name="text">
                <string>--</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>传输函数来源</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="2">
           <item>
            <widget class="QComboBox" name="comboBox_TFSrc">
             <item>
              <property name="text">
               <string>无</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>文件读入/自定义</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>蓝红渐变</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_TF">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>传输函数</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,2">
          <item>
           <widget class="QPushButton" name="pushButton_ImportTF">
            <property name="text">
             <string>导入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_ImportedTF">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_Import">
       <property name="title">
        <string>导入</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>尺寸信息（可选）</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2,1,2,1,2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>X尺寸</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_DimX">
               <property name="maximum">
                <number>1024</number>
               </property>
               <property name="singleStep">
                <number>10</number>
               </property>
               <property name="value">
                <number>256</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Y尺寸</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_DimY">
               <property name="maximum">
                <number>1024</number>
               </property>
               <property name="singleStep">
                <number>10</number>
               </property>
               <property name="value">
                <number>256</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Z尺寸</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_DimZ">
               <property name="maximum">
                <number>1024</number>
               </property>
               <property name="singleStep">
                <number>10</number>
               </property>
               <property name="value">
                <number>256</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_9">
          <property name="title">
           <string>归一化用数据范围（可选）</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="0,1,2,1,2">
           <item>
            <widget class="QCheckBox" name="checkBox_UseValRngForNorm">
             <property name="text">
              <string>使用</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>最小值</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_MinValForNorm">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="minimum">
              <double>-99999999.000000000000000</double>
             </property>
             <property name="maximum">
              <double>99999999.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>最大值</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_MaxValForNorm">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="minimum">
              <double>-99999999.000000000000000</double>
             </property>
             <property name="maximum">
              <double>99999999.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>RAW（uint8）格式体数据</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
             <item>
              <widget class="QPushButton" name="pushButton_ImportRAW">
               <property name="text">
                <string>导入</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_ImportedRAW">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>TXT格式体数据</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>无效值</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_NullVal">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="minimum">
                <double>-99999999.000000000000000</double>
               </property>
               <property name="maximum">
                <double>99999999.000000000000000</double>
               </property>
               <property name="value">
                <double>-9999.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
             <item>
              <widget class="QPushButton" name="pushButton_ImportTXT">
               <property name="text">
                <string>导入</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_ImportedTXT">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Labeled TXT格式体数据</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
             <item>
              <widget class="QPushButton" name="pushButton_ImportLabeledTXT">
               <property name="text">
                <string>导入</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_ImportedLabeledTXT">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,2">
             <item>
              <widget class="QPushButton" name="pushButton_ImportLabeledTXTTimeSeries">
               <property name="text">
                <string>导入时序数据</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_ImportedLabeledTXTTimeSeries">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_10">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>读入信息</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>尺寸</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_ImportedDim">
               <property name="text">
                <string>--, --, --</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>数据范围</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_ImportedValRng">
               <property name="text">
                <string>[--, --]</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>经纬高范围</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_ImportedLLHRng">
               <property name="text">
                <string>[--, --], [--, --], [--, --]</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_Export">
       <property name="title">
        <string>导出</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="pushButton_ExportVolume">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>导出RAW格式
体数据（uint8）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_ExportTF">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>导出传输函数</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
